SimSim/AppDelegate.m

```diff
@@ -139,7 +139,12 @@ - (void) addSubMenusToItem:(NSMenuItem*)item usingPath:(NSString*)path
         [[NSMenuItem alloc] initWithTitle:@"Finder" action:@selector(openInFinder:) keyEquivalent:@"2"];
         [finder setRepresentedObject:path];
         [subMenu addItem:finder];
-        
+
+        NSMenuItem* iTerm =
+        [[NSMenuItem alloc] initWithTitle:@"iTerm" action:@selector(openIniTerm:) keyEquivalent:@"3"];
+        [iTerm setRepresentedObject:path];
+        [subMenu addItem:iTerm];
```

Comment:
You need to check if iTerm is available first. It makes no sense to show this menu item to ones who don't use it.